modules/globebrowsing/src/tileprovider/ffmpegtileprovider.cpp

```diff
@@ -0,0 +1,434 @@
+/*****************************************************************************************
```

Comment:
Again, I suggest to remove all tileprovider stuff from this PR. Skipping these files for now